### PR TITLE
Removed leftover remove operation

### PR DESCRIPTION
**Pull request by lalaithan**
Removed leftover remove operation mentioned in issue #51

### CODE REVIEW

File: src/Config_Command.php [hunk 1]
```diff
@@ -549,9 +549,6 @@ public function delete( $args, $assoc_args ) {
 						WP_CLI::error( "The {$type} '{$name}' is not defined in the 'wp-config.php' file." );
 					}
 			}
-
-			$config_transformer->remove( $type, $name );
```
Comment (schlessera) [inline src/Config_Command.php]: You've removed the wrong `remove`.

Comment (schlessera) [inline src/Config_Command.php]: You've removed the wrong `remove`.

The one that is superfluous is at https://github.com/wp-cli/config-command/pull/52/files#diff-a3ce20ba86d7a7fcd6d399db066ca679R614

Comment (lalaithan) [reply src/Config_Command.php]: You're right, I see the error in my own remove. Sorry.

Comment (schlessera) [reply src/Config_Command.php]: No need to be sorry, it was an easy mistake and that's what code reviews are for.

Comment (schlessera) [reply src/Config_Command.php]: @lalaithan Let me know if you have trouble updating this pull request.

**Comment by schlessera** [comment]
Thanks for the pull request, @lalaithan !

**Pull request merged**